server/storage/types.go

```diff
@@ -34,9 +34,11 @@ const (
 const (
 	ShardRoleLeader ShardRole = iota + 1
 	ShardRoleFollower
+)
 
-	ShardStatusPartitionOpen ShardStatus = iota + 1
-	ShardStatusReady
+const (
+	ShardStatusReady ShardStatus = iota + 1
```

Comment:
Why it starts from 1? Is it ok to start form zero?

Reply:
The default value in Golang is 0, starting from 1 there will be no ambiguity.